packages/core/test-helpers/core-test-helpers-test-utils/src/create_hidden_type_variants.ts

```diff
@@ -8,11 +8,15 @@
 
 import { SavedObjectsType } from '@kbn/core-saved-objects-server';
 
+type SavedObjectsTypeWithManaged = SavedObjectsType & {
+  managed?: boolean;
+};
+
 export const createHiddenTypeVariants = (createOptions: {
   name: string;
   hide?: boolean;
   hideFromHttpApis?: boolean;
-}): SavedObjectsType => {
+}): SavedObjectsTypeWithManaged => {
```

Comment:
I'm not sure to get this change, `createHiddenTypeVariants` is supposed to build SO type definitions, why do we need to add a `managed` property to the type of the return value here?

I assume it's related to the change in `src/core/server/integration_tests/saved_objects/routes/bulk_create.test.ts`, but I'm still not sure to get why this was necessary?